SADL/AM-CDM-part.sadl

```diff
@@ -0,0 +1,27 @@
+uri "http://kdl.ge.com/AM-CDM-part" alias AM-CDM-part version "$Revision:$ Last modified on   $Date:$".
+
+Part (note "Instances of part designs in the as-built state.") is a MaterialStock,
```

Comment:
Add an OP "project" of "Project"

Reply:
I removed the parts object property from Project and added a project object property in Part.